microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -204,68 +193,125 @@ Adds the **CSP Violations** tool, which displays any Content Security Policy (CS
 
 ![The CSP Violations tool.](index-images/csp-violations-tool.png)
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
 
 
 <!-- ====================================================================== -->
 ## Record coverage while performance tracing
 
 Records coverage while performance tracing.
-<!-- needs content, 0 hits in fts in this repo -->
+<!-- todo: need text -->
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
 
 
 <!-- ====================================================================== -->
 ## Show option to take heap snapshot where globals are treated as root
+<!-- older checkbox -->
 
 Shows the option to take a heap snapshot where globals are treated as root.
-<!-- needs content -->
+<!-- todo: need text, except going away -->
+
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is not present in Microsoft Edge Canary v105.
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+See also:
+* [Record heap snapshots using the Memory tool](../memory-problems/heap-snapshots.md)
 
 
 <!-- ====================================================================== -->
 ## Show back/forward cache blocking reasons in the frame tree structure view
+<!-- older checkbox -->
 
 Whether to show back/forward cache blocking reasons in the frame tree structure view.
-<!-- needs content -->
+<!-- todo: need text, except going away -->
+
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is not present in Microsoft Edge Canary v105.
 
-*  This checkbox is present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+See also:
+* [Application tool, to manage storage](../storage/application-tool.md)
+
+
+<!-- ====================================================================== -->
+## Show option to expose internals in heap snapshots
```

Comment:
@captainbrosset  @zoherghadyali - need text for v105 Experiments checkbox "Show option to expose internals in heap snapshots"